lib/encryption/index.js

```diff
@@ -0,0 +1,7 @@
+const { encryptString } = require('./encrypt')
```

Comment:
using this syntax allows you to pull specific methods you want from the `encrypt` file into variables rather than require the full file